<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>숙박 어플 사용여부 관리(appcomp_use_info.py)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>stay_icon_mnt.png</normaloff>stay_icon_mnt.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 220, 220);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1581</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font: 12pt &quot;Arial&quot;;
    border: 1px solid silver;
    border-radius: 1px;
    margin-top: 9px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 0px 5px 0px 5px;
}</string>
   </property>
   <property name="title">
    <string>숙박 어플 업체명</string>
   </property>
   <widget class="QLineEdit" name="ed_homepage">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>59</y>
      <width>1321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: black;
font: 12pt &quot;Arial&quot;;
 border: 1px solid rgb(177, 177, 177);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>62</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>홈페이지 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>업  체  명 :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_comp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>27</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: black;
font: 12pt &quot;Arial&quot;;
 border: 1px solid rgb(177, 177, 177);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_clear">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>81</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>현 자료를 저장합니다.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
background-color: rgba(123, 141, 255, 255);
}

QPushButton:hover{
color: rgba(123, 141, 255, 255);
background-color: white;
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_save">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>24</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="toolTip">
     <string>현 자료를 저장합니다.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: white;
background-color: rgba(123, 141, 255, 255);
}

QPushButton:hover{
color:blue;
background-color: rgba(255, 255, 60, 255);
}

QPushButton:pressed {
color: red;
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_astel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>519</x>
      <y>26</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: black;
font: 12pt &quot;Arial&quot;;
 border: 1px solid rgb(177, 177, 177);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>29</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>대표번호 :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_use">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>97</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>숙박 어플 사용</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tb_internet">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>58</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>인터넷 연결</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>internet.png</normaloff>internet.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tw_appcomp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>1581</width>
     <height>755</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
background-color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;
}</string>
   </property>
   <column>
    <property name="text">
     <string>사용여부</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>업체명</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>대표번호</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>홈페이지 주소</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
